dbms/src/DataStreams/UnorderedTransformAction.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <DataStreams/UnorderedTransformAction.h>
+
+namespace DB
+{
+Block UnorderedTransformAction::getHeader() const
+{
+    return header;
+}
+
+bool UnorderedTransformAction::transform(Block & block)
```

Comment:
can we reuse this code for `DMSegmentThreadInputStream`?
https://github.com/pingcap/tiflash/blob/dc194135de55b3b88c348215d24d7f173e9f055b/dbms/src/Storages/DeltaMerge/DMSegmentThreadInputStream.h#L111-L121